rohmu/object_storage/base.py

```diff
@@ -108,7 +108,7 @@ def _should_multipart(
             # We could actually sniff from fd if it is seekable; left TODO for now.
             return default
 
-        return size > chunk_size
+        return int(size) > chunk_size
```

Comment:
This is the wrong place to do it and against type hint. It should be fixed in source (wherever that is).